test/lib/rules/prevent-use-of-window-location-test.js

```diff
@@ -1,25 +0,0 @@
-const rule = require('../../../lib/rules/prevent-use-of-window-location');
-const RuleTester = require('eslint').RuleTester;
-
-const ruleTester = new RuleTester();
-
-ruleTester.run('prevent-window-location-assignment', rule, {
```

Comment:
I think we still want the test since the rule still lives in this repo, right?

Reply:
https://github.com/Root-App/root-eslint-plugin/pull/13/files/d166e13c4d018b0a28154e88592fa783efefbfbb#diff-a3ad2d4385789e3bf9673d381ffcbf1d this PR removes it

Reply:
It doesn't remove it- it just stops defaulting it to error on every project. The rule still exists at `lib/rules/prevent-use-of-window-location`

Reply:
(to be clear, we do want the rule's code to exist here. We just don't want to opt-in rather than out)